App/features/cafes/sagas.ts

```diff
@@ -134,13 +134,41 @@ function* refreshExpiredSessions() {
   }
 }
 
+function* migrateUSW() {
+  try {
+    // Old us-west
+    const usw = '12D3KooWSsM117bNw6yu1auMfNqeu59578Bct5V4S9fWxavogrsw'
+    // New us-west
+    const repl = '12D3KooWSdGmRz5JQidqrtmiPGVHkStXpbSAMnbCcW8abq6zuiDP'
+    const cafes: Cafes = yield select((state: RootState) => state.cafes.cafes)
```

Comment:
Sorry for chiming in after this has been merged, but I see a problem here. This saga is being called on `actions.getCafeSessions.request`. So at the time it's called, the async call to get the cafe data is in flight, but your're querying the redux state for the cafes here. It seems like that state won't be populated until a `actions.getCafeSessions.success` happens.

Reply:
but will cafes be populated if sessions are invalid? i had a mini-thread about this with myself on here yesterday and it's why i changed it from success => request, my concern was someone coming online in 2 months and having invalid sessions but still attempting to use the old us-west

Reply:
okay, if my hypothesis above is right, and the cafes object isn't populated without a session, then i can configure this to run with success or failure of sessions

Reply:
I don't think the node knows anything about if the sessions are valid or not. 

The way this works now, I don't see you'd ever detect any cafe session. Seems like they'd always be empty since that request is in flight. It would only work on a subsequent run of the saga, not the first one. 

You can also just remove the dependency on other redux and sagas: This saga could run once when the main saga is created and just call the `Textile.cafes.sessions` directly to get the cafe sessions.